docs/controls/AdaptiveGridView.md

```diff
@@ -10,23 +9,100 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, AdaptiveGridView,
 
 The **AdaptiveGridView Control** presents items in a evenly-spaced set of columns to fill the total available display space. It reacts to changes in the layout as well as the content so it can adapt to different form factors automatically.
 
-If there are not enough items to fill one row, the control will stretch the items until all available space is filled. This can result in much wider items than specified. If you prefer your items to always stay close to the DesiredWidth, you can set the **StretchContentForSingleRow** property to **false**, to prevent further stretching.
+There are 3 ways to use this Control. 
+
+* You can set `DesiredWidth` and `ItemHeight`, which will scale the **width** of each item, and adjust the number of columns on demand, using all horizontal space.
+
+* To maintain aspect ratio, You can set `DesiredWidth` only. This will allow the **height** and **width** of each item, and adjust the number of columns on demand, using all horizontal space.
+
+* Using `OneRowModeEnabled`, you can set `DesiredWidth` and `ItemHeight`, and the **width** of each item, and the number of visible columns will be adjusted on demand, using all horizontal space.
 
 ## Syntax
 
 ```xaml
-
 <controls:AdaptiveGridView  Name="AdaptiveGridViewControl"
           ItemHeight="200"
           DesiredWidth="300"
           ItemTemplate="{StaticResource PhotosTemplate}">
 </controls:AdaptiveGridView>
-
 ```
 
-## Example Image
```

Comment:
Don't remove `## Example Image` section. This section will give quick glace of the feature.

Reply:
I didn't remove it, it is under the first Use case.

Reply:
I know but `## Example Image` section is a standard section for every control. Removing it from this control is not nice. Maybe move the image from "the first Use case" to `Example Image`.

Reply:
The image is only relevant to one usage scenario. It doesn't present the other scenarios.

Reply:
Except that these different scenarios pretty much change the whole presentation of the control.

Reply:
The default control image should still be in the "normal section"